base/filesystem.jl

```diff
@@ -192,6 +216,18 @@ function position(f::File)
     return ret
 end
 
+function eof(f::File)
+    a = Vector{UInt8}(1)
+    ret = ccall(:jl_fs_read, Int32, (Int32, Ptr{Void}, Csize_t), f.handle, a, 1)
+    systemerror("eof", ret == -1)
+    if ret == 1
+        skip(f, -1)
```

Comment:
> I've replaced it with `eof(f::File) = nb_available(f) == 0`.

Without deleting this version of the exact same method. One of them is redundant.